packages/cli/mixin.core.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+const { format } = require('url');
+const prettyMS = require('pretty-ms');
+
+class LoggerPlugin {
+  apply(compiler) {
+    compiler.hooks.done.tap('hops-logger-plugin', stats => {
+      if (stats.compilation.name === 'develop') {
+        console.log(
+          'Compiled successfully',
+          'in',
+          prettyMS(stats.endTime - stats.startTime)
+        );
+      }
+    });
+  }
+}
+
+module.exports = class CLIMixin {
```

Comment:
We could add `extends Mixin` here to make it clear(er) it's a mixin.

That would also be good in order to not break as soon as it is actually required to inherit from Mixin.
And thats what we are doing anywhere else.

Reply:
I don't think we can - at least not while the mixin is still part of `@untool/core`, because then the CLI (which might be installed globally) will have a dependency to `@untool/core`.

But what we could do, is to make the `Mixin` a separate package in untool and then we could add it (or rather `hops-mixin`) as a dependency to the CLI package.

/cc @dmbch

Reply:
As the `Mixin` base class is just a five-liner, I am opposed to releasing it separately. I personally think that for this very use-case, it is completely fine to have a standalone class.